docs/tutorial/index.md

```diff
@@ -9,65 +9,67 @@ Gatsby Seite basiered auf einer Starter Vorlage zu leiten. Während wir diesen
 Prozess durchführen werden wir einige allgemeinere Webentwicklungsthemen
 einführen und die unterliegende Struktur einer Gatsby Seite erläutern.
 
-> Das gesamte Tutorial ist dazu gedacht so zugänglich wie nur möglich zu sein,
-für Personen, die (noch!) keine Webentwicklungserfahrung haben - es ist nicht
-notwendig ein Experte zu sein. Wenn du lieber direkt zum Code springen möchtest,
-dann kannst du dieses Schritt-für-Schritt Tutorial überspringen und dir die
-[Schnell Start](/docs/quick-start) Seite anschauen.
+> Das gesamte Tutorial ist für für Personen gedacht, die (noch!) keine
+> Webentwicklungserfahrung haben - es ist nicht notwendig ein Experte zu sein.
+> Wenn du lieber direkt zum Code springen möchtest, dann kannst du dieses
+> Tutorial überspringen und dir die [Schnell Start](/docs/quick-start) Seite
+> anschauen.
 
 ## Gatsby Grundlagen
 
 0. [Setz deine eigene Entwicklungsumgebung auf](/tutorial/part-zero/): Wir
-   führen dich in die Kerntechnologien, die Gatsby betreiben, ein und leiten
-   dich durch das Aufsetzen deiner Entwicklungsumgebung.
-1. [Lerne die Gatsby Bausteine kennen](/tutorial/part-one/): Der Beginn neuer
-   Projekte, das Entwickeln und Veröffentlichen von Webseiten.
-2. [Einführung in die Nutzung von CSS in Gatsby](/tutorial/part-two/): Entdecke
+   setzen deine Entwicklungsumgebung auf und erläutern die Kerntechnologien von
+   Gatsby.
+1. [Lerne die Gatsby Bausteine kennen](/tutorial/part-one/): Wir erklären,
+   wie du neue Gatsby Projekte startest, sie entwickelst und als Seite
+   veröffentlichst.
+1. [Einführung in die Nutzung von CSS in Gatsby](/tutorial/part-two/): Entdecke
    Bibliotheken wie Typography.js und CSS Module.
-3. [Bauen von verschachtelten Layouts in Gatsby](/tutorial/part-three/):
+1. [Bauen von verschachtelten Layouts in Gatsby](/tutorial/part-three/):
    Layouts sind Sektionen deiner Seite, die über verschiedene Seiten wieder
    benutzt werden, wie Kopf- und Fußzeilen.
 
 ## Fortgeschrittene Tutorials
 
-In diesen fortgeschrittenen Tutorials, wirst du lernen wie du Daten von nahezu
-überall mit Hilfe von GraphQL in deine Gatsby Seite integrieren kannst.
+In diesen fortgeschrittenen Tutorials lernst du, wie Daten von nahezu
+überall mit Hilfe von GraphQL in eine Gatsby Seite integriert werden können.
 
 4. [Abfragen von Daten in einem Blog](/tutorial/part-four/): Erstellung eines
-   Blogs und Nutzung einer GraphQL Anfrage um deinen Webseiten Titel in die
-   Kopfzeile des Blogs zu ziehen.
+   Blogs und Nutzung einer GraphQL Abfrage um deinen Webseiten Titel in der
+   Kopfzeile deines Blogs darzustellen.
```

Comment:
```suggestion
   Kopfzeile darzustellen.
```

It's a bit redundant, what do you think?

Reply:
I am not sureI am unsure. It seems redundant, but it like the reading flow this one actually more. And it is clarification without completly obtrusing the reading flow.